numpy/core/src/multiarray/shape.c

```diff
@@ -87,7 +87,14 @@ PyArray_Resize(PyArrayObject *self, PyArray_Dims *newshape, int refcheck,
         }
 
         if (refcheck) {
+#ifdef PYPY_VERSION
+            PyErr_SetString(PyExc_ValueError,
+                    "cannot resize an array on PyPy."\
+                    "  Use the resize function");
```

Comment:
Nitpick, but you could point to `refcheck=False` here? Though I guess it is also giving people a gun and carefully pointing it at their feet, hmmm. (that backslash seems unnecessary as well ;)).